packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -137,12 +140,11 @@ export function PrivateBlockToolbar( {
 			isUsingBindings: _isUsingBindings,
 			hasParentPattern: _hasParentPattern,
 			hasContentOnlyLocking: _hasTemplateLock,
-			showShuffleButton: isZoomOut(),
-			showSlots: ! isZoomOut(),
-			showGroupButtons: ! isZoomOut(),
-			showLockButtons: ! isZoomOut(),
```

Comment:
I prefer show booleans rather than zoom-out, because it makes the intent clearer. That said, I know that we were kind of using a mix here which is not great either.

Reply:
You can still do that outside of the selector. Adding more keys makes this selector less performant :D

Reply:
not really if the selector return value changes at the same time but yeah it's easier to do perf mistakes like that I agree.